src/applications/disability-benefits/686/config/form.js

```diff
@@ -121,81 +124,74 @@ const formConfig = {
     vaFileNumber
   },
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          path: 'veteran-information',
-          title: 'Veteran Information',
+        applicantInformation: {
+          title: 'Applicant information',
```

Comment:
Information should be capitalized, I think.

Reply:
k